### DISCUSSION TITLE
Delay in Binary Outputs

### DISCUSSION THREAD

**Discussion by lxysl** [RavenSystem/esp-homekit-devices#1743] [MEPLHAA Scripts]
hello, I want to switch two GPIOs with 1 second delay between them. For example, switch GPIO 0 from low level to high level first, and then switch  GPIO 2 from high level to low level after 1 second. Is there anyway to do that? Thanks!

**Comment by clint80** [comment]
hello I integrated a timer in the json of my garage door that once closed, actuate the security with
{"I": 5, "0": {"M": [[3,1]]}}. At the end of 5s Change Status of my 3rd relay

**Comment by i3laze** [comment]
You should add a dummy inching accessory and use internal notifications.
GPIO1 high -> notify 1sec inching switch on
Inching switch off -> notify GPIO2


@RavenSystem, I feel there’s a demand for delayed actions.. 
Could you please consider to introduce some delays? Maybe within notifications.

**Comment by RavenSystem** [comment]
Use a switch outside HK to use delayed actions.

**Comment by RavenSystem** [comment]
Please, use Discussions section and leave Issues section only for issues. Thanks!

https://github.com/RavenSystem/esp-homekit-devices/wiki#issues

**Comment by RavenSystem** [comment]
https://github.com/RavenSystem/esp-homekit-devices/discussions/1748
